tests/testthat/test-datatype.R

```diff
@@ -224,12 +224,18 @@ test_that("contains_* functions for datatype work", {
 })
 
 test_that("Enum", {
-  expect_identical(
-    as_polars_series(c("z", "z", "k", "a"))$
-      cast(pl$Enum(c("z", "k", "a")))$
-      to_r(),
-    factor(c("z", "z", "k", "a"))
-  )
+  # Generate 100 random data frames with random
+  # orders for levels. They should always match
```

Comment:
Please use quickcheck or patrick instead of for loop.
https://www.etiennebacher.com/posts/2024-10-01-using-property-testing-in-r/